packages/backend-common/src/sqs.ts

```diff
@@ -105,7 +111,12 @@ export const generateOutputSignedUrlAndSendMessage = async (
 		return await sendMessage(
 			client,
 			queueUrl,
-			JSON.stringify({ ...job, translate: true }),
+			JSON.stringify({
+				...job,
+				id: `${s3Key}-translation`,
```

Comment:
while we're at it it feels weird to define this here and in [`generateOutputSignedUrls`](https://github.com/guardian/transcription-service/pull/125/files#diff-7a6f98c31cdd871b93d5d3354f444d5d94916552fd05005856cdf0d460239415R318). Might be cleaner to pass filename to that function?

Reply:
good shout, definitely want to minimise the number of places we're changing the id to reduce problems exactly like this one